fingerprint/infra/nec-bio-sdk/src/test/java/com/simprints/fingerprint/infra/necsdkimpl/acquisition/template/FingerprintTemplateProviderImplTest.kt

```diff
@@ -1,5 +1,6 @@
 package com.simprints.fingerprint.infra.necsdkimpl.acquisition.template
 
+import SecuGen.FDxSDKPro.WSQConverter
```

Comment:
I did not notice it in the actual project, but this namespace breaks all kinds of naming conventions. Looks almost like .Net or Objective-C import :D

Reply:
Do we need to have all wrapper classes in that package, tho? Maybe we could have an explicit separation - correct package name for wrapper, forced package name for the JNI parts?

Reply:
Done